egs2/l3das22/enh1/pyscripts

```diff
@@ -0,0 +1 @@
+../../TEMPLATE/asr1/pyscripts
```

Comment:
though may not be different, by use `enh1/pyscripts` would be better (just in case future differences)

Reply:
Changed.